scpd/lms/static/sass/partials/base/_variables.scss

```diff
@@ -0,0 +1,13 @@
+// lms - custom variables
```

Comment:
I don't think these comments are particularly helpful.
We're in a file called `_variables` within the LMS directory, so it's pretty clear that these are LMS variables.

Similarly below, the code says `import [...] variables [...]`, so it is also pretty clear that it's `Import[ing] [...] variables`